rosdep/python.yaml

```diff
@@ -5321,6 +5321,13 @@ python-tqdm:
     xenial:
       pip:
         packages: [tqdm]
+python3-transformers-pip:
```

Comment:
This is failing CI because this needs to be after the `python3-transforms3d-pip` key.

Reply:
Oh sorry, fixed it.